qa/suites/orch/cephadm/smoke-roleless/1-start.yaml

```diff
@@ -1,3 +1,8 @@
+overrides:
+  ceph:
+    log-ignorelist:
+      - MON_DOWN
+      - mons down, quorum
```

Comment:
need to add `MDS_ALL_DOWN` for the nfs tests that setup a cephfs. If the MDS actually stay down I don't think the mount will work properly, so this error is probably happening temporarily during setup.